erts/emulator/beam/erl_bits_f16.h

```diff
@@ -0,0 +1,193 @@
+/*
+ * C-implementation of Float16 - https://github.com/Maratyszcza/FP16
+ *
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2017 Facebook Inc.
+ * Copyright (c) 2017 Georgia Institute of Technology
+ * Copyright 2019 Google LLC
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy of this
+ * software and associated documentation files (the "Software"), to deal in the Software
+ * without restriction, including without limitation the rights to use, copy, modify,
+ * merge, publish, distribute, sublicense, and/or sell copies of the Software, and to
+ * permit persons to whom the Software is furnished to do so, subject to the following
+ * conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all copies
+ * or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+ * INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR
+ * PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE
+ * FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR
+ * OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+ * DEALINGS IN THE SOFTWARE.
+ */
+static ERTS_INLINE float fp32_from_bits(uint32_t w) {
```

Comment:
Sorry about the back and forth, but we need to change all instances of `uint??_t` in this file too. :(

Reply:
No problem! I should probably get a Windows machine next time I am working on this part of ERTS. :)

I have done the requested changes. I have also added a short notice at the top of this file (below the copyright disclaimer) to list all of the modifications we have done. This will make it easier to track changes or update the code in the future.